openshift/cloud-deploy/deploy.sh

```diff
@@ -3,7 +3,9 @@
 # We use $PTH_ENV in ../deploy.sh
 source conf
 
-${CLOUD_DEPLOYER_PATH}/deploy.sh --conf conf --services services --secrets-template secrets-template-aws.yaml --secrets-deploy ../secrets-deploy.sh --system-deploy ../deploy.sh
-
+# We need to deploy these secrets first
 curl -O https://gitlab.cee.redhat.com/bayesian/secrets/raw/master/secrets-template.yaml
 ../secrets-deploy.sh --secrets-file secrets-template.yaml
+
+${CLOUD_DEPLOYER_PATH}/deploy.sh --conf conf --services services --secrets-template secrets-template-aws.yaml --secrets-deploy ../secrets-deploy.sh --system-deploy ../deploy.sh
```

Comment:
This script is failing

```
[cloud-deploy] $ ./deploy.sh 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   926  100   926    0     0   3398      0 --:--:-- --:--:-- --:--:--  3404
secret "worker" created
secret "jobs" created
./deploy.sh: line 10: /deploy.sh: No such file or directory

```

Apparently the environment variable `CLOUD_DEPLOYER_PATH` is missing, but I am not sure what the value for this should be. It is definitely not present in `conf` file.